src/components/CMessageRow.vue

```diff
@@ -82,21 +86,26 @@ export default {
       if (!this.startedDatetime || !this.enqueuedDatetime) {
         return null;
       }
-      return formatDistance(this.enqueuedDatetime, this.startedDatetime);
+      const diff = utils.getDistance(this.startedDatetime, this.enqueuedDatetime);
+      return `${diff.hours}:${diff.minutes}:${diff.seconds}`;
     },
-
-    executionTime() {
-      if (!this.startedDatetime) {
+    remainingTime() {
+      if (this.endDatetime) {
```

Comment:
some weird things happens when state is pending
![image](https://user-images.githubusercontent.com/6851979/84910351-11a8cb80-b0b7-11ea-9ac8-21ee863eeb15.png)

`if (this.endDatetime || !this.startedDatetime) { 
   return null
}`